azurerm/resource_arm_app_service_plan.go

```diff
@@ -136,12 +139,16 @@ func resourceArmAppServicePlan() *schema.Resource {
 				ConflictsWith: []string{"properties.0.reserved"},
 			},
 
-			"maximum_number_of_workers": {
+			"maximum_elastic_worker_count": {
 				Type:     schema.TypeInt,
+				Optional: true,
 				Computed: true,
 			},
 
-			"tags": tagsSchema(),
+			"maximum_number_of_workers": {
```

Comment:
Could we add this to a test? perhaps the `TestAccAzureRMAppServicePlan_completeWindows` one

Reply:
Added to `TestAccAzureRMAppServicePlan_premiumConsumptionPlan` as it was the correct Kind and SKU